upgrade/upgrade.go

```diff
@@ -104,3 +118,100 @@ func ClaimSwapPoolsYield(
 
 	return nil
 }
+
+// InitializeHyperlaneModule creates a default Hyperlane ISM and Mailbox.
+func InitializeHyperlaneModule(
+	ctx context.Context,
+	logger log.Logger,
+	addressCodec address.Codec,
+	hyperlaneKeeper *hyperlanekeeper.Keeper,
+) error {
+	chainId := sdk.UnwrapSDKContext(ctx).ChainID()
+
+	var localDomain uint32
+	switch chainId {
+	case TestnetChainID:
+		localDomain = TestnetHyperlaneDomain
+	case MainnetChainID:
+		localDomain = MainnetHyperlaneDomain
+	default:
+		return fmt.Errorf("cannot initialize hyperlane module on %s chain", chainId)
+	}
+
+	authority, err := addressCodec.BytesToString(authoritytypes.ModuleAddress)
+	if err != nil {
+		return errors.New("unable to encode authority address")
+	}
+
+	ismServer := ismkeeper.NewMsgServerImpl(&hyperlaneKeeper.IsmKeeper)
+	pdhServer := pdhkeeper.NewMsgServerImpl(&hyperlaneKeeper.PostDispatchKeeper)
+	hyperlaneServer := hyperlanekeeper.NewMsgServerImpl(hyperlaneKeeper)
+
+	createRoutingIsmRes, err := ismServer.CreateRoutingIsm(ctx, &ismtypes.MsgCreateRoutingIsm{Creator: authority})
+	if err != nil {
+		return fmt.Errorf("unable to create routing ism: %w", err)
+	}
+	ismId := createRoutingIsmRes.Id
+	logger.Info("created noble hyperlane ism", "id", ismId)
+
+	if isms, found := HyperlaneDefaultISMs[chainId]; found {
+		for _, ism := range isms {
+			sort.Strings(ism.Validators)
```

Comment:
why do we need to do this sort here?

Reply:
It's required when creating the ISM: https://github.com/bcp-innovations/hyperlane-cosmos/blob/main/x/core/01_interchain_security/types/multisig.go#L67-L70.

I wanted to maintain the ordering from the documentation, which is why it's not sorted in the constant.

But happy to update!

Reply:
Not necessary! I was just curious because it felt very intentional so I figured there was a good reason for it.